static/js/settings/languages.js

```diff
@@ -0,0 +1,52 @@
+/*global each, category_template, $categories_form */
```

Comment:
Codacy found an issue: ['category_template' is defined but never used.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6225754)